source/MRMesh/MRMeshBuilder.h

```diff
@@ -45,7 +45,7 @@ struct VertDuplication
 // resolve non-manifold vertices by creating duplicate vertices in the triangulation (which is modified)
 // return number of duplicated vertices
 MRMESH_API size_t duplicateNonManifoldVertices( Triangulation & t, FaceBitSet * region = nullptr,
-    std::vector<VertDuplication>* dups = nullptr );
+    std::vector<VertDuplication>* dups = nullptr, VertId lastValidVert = {} );
```

Comment:
Please add comment that `lastValidVert` needs to be specified only if `region` contains not whole mesh. And the first duplicated vertex will have `id=lastValidVert+1`